### ISSUE TITLE
[TASK][BETA 1.0] docker image

### ISSUE THREAD

**Issue by Robonau** [JiPaix/Fukayo#452]
**Is your feature request related to a problem? Please describe.**
i want to run it on a headless server (that is already setup for containers)

**Describe the solution you'd like**
ideally github actions setup to automatically build and test a docker image on every release and preview(like beta release)

**Describe alternatives you've considered**
none

**Additional context**
[how techidesk does it](https://github.com/Suwayomi/docker-tachidesk) (.github/workflows/)

**Comment by JiPaix** [comment]
No idea how docker works but it's worth looking at it as Fukayo should be able to run in headless mode with `useFork()`
https://github.com/JiPaix/Fukayo/blob/92cff460aec56d2d032eaad91d23ec74992604e6/packages/api/src/index.ts#L13
https://github.com/JiPaix/Fukayo/blob/92cff460aec56d2d032eaad91d23ec74992604e6/packages/api/src/types/index.ts#L1-L59

The API relies on puppeteer, [xvfb](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml) will be required:
```cmd
Xvfb :100
export DISPLAY=:100.0
node ./script/that/use/useFork.js
```
[xvfb-run](https://github.com/revnode/xvfb-run) might work too
```cmd
xvfb-run node ./script/that/use/useFork.js
```

**Comment by Robonau** [comment]
as long as ik how it can be run in headless i can make a dockerfile that i can build
but ideally it would be an image built by github actions on every release (and beta release) (idk github actions though)

**Comment by JiPaix** [comment]
closed by 47eaadb8e48079a8dcbe6e156fdbe561979b6bf4
